lib/BuildSystem/BuildSystem.cpp

```diff
@@ -2853,12 +2853,30 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     return ExternalCommand::configureAttribute(ctx, name, values);
   }
 
+  template <typename String>
+  std::string escapeForJson(String const& in) const {
+    std::ostringstream ss;
+    for (auto i = in.begin(); i != in.end(); i++) {
+      switch (*i) {
+      //FIXME: We need to properly escape unicode sequences.
+      case '\\': ss << "\\\\"; break;
+      case '"': ss << "\\\""; break;
+      case '/': ss << "\\/"; break;
```

Comment:
Don't escape / as it is not required and it's very annoying for readability with file paths